### PR TITLE
Wrapper bricks

### PR DESCRIPTION
**Pull request by vdumoulin**
Solves #401.

### CODE REVIEW

File: blocks/bricks/wrapper.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    application_method : callable
+        Some brick's :meth:`apply` method
+
+    """
+    def __init__(self, application_method, **kwargs):
+        super(As2D, self).__init__(**kwargs)
+        self.application_method = application_method
+        self.children = [self.application_method.brick]
+
+    @application(inputs=['input_'], outputs=['output'])
```
Comment (rizar) [inline blocks/bricks/wrapper.py]: When a brick is one input and one output they are typically never used. So I would add it later if it turns out necessary.

**Comment by rizar** [comment]
Everything looks nice! `WithAxesSwapped` would be also nice to have.

File: blocks/bricks/wrapper.py (hunk 1 above)
Comment (bartvm) [inline blocks/bricks/wrapper.py]: Really? I would have added them myself as well, if for nothing else, then for consistency.

Comment (rizar) [inline blocks/bricks/wrapper.py]: Hmm, I have just seen that someone has already added lot's of trivial input and output names for many bricks... could it be you? :)

**Comment by rizar** [comment]
Would anybody mind moving `bricks/wrapper.py` to `bricks/wrappers.py` or `bricks/wrap.py`?

File: blocks/bricks/wrapper.py (hunk 1 above)
Comment (bartvm) [inline blocks/bricks/wrapper.py]: I guess? Not in any conscious effort though, I probably just figured it was more consistent.

**Comment by bartvm** [comment]
No objections, although I'd prefer `wrappers.py`

**Comment by vdumoulin** [comment]
@rizar @bartvm I'll implement `WithAxesSwapped` and rename the module to `wrappers` in the same pull request.

**Comment by vdumoulin** [comment]
Just found out the renaming has already been done, so I'll just implement `WithAxesSwapped`.

**Comment by rizar** [comment]
I guess you should open a new one now.

**Pull request merged**